<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSettingsDialog</class>
 <widget class="QDialog" name="DeviceSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Parameter Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMachining">
      <attribute name="title">
       <string>Machining</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Cutting</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Start Speed(mm/s)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="EditSlider" name="editSliderCuttingStartSpeed" native="true"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Run Speed(mm/s)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="EditSlider" name="editSliderCuttingRunSpeed" native="true"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Power</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="EditSlider" name="editSliderCuttingPower" native="true"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Start/Power</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="EditSlider" name="editSliderCuttingStartPowerRate" native="true"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Run/Power</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="EditSlider" name="editSliderCuttingRunPowerRate" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Engraving</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Start Speed(mm/s)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="EditSlider" name="editSliderEngravingStartSpeed" native="true"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Run Speed(mm/s)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="EditSlider" name="editSliderEngravingRunSpeed" native="true"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Power</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="EditSlider" name="editSliderEngravingPower" native="true"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Start/Power</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="EditSlider" name="editSliderEngravingStartPowerRate" native="true"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Run/Power</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="EditSlider" name="editSliderEngravingRunPowerRate" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Cutting tune-up</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Power</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="EditSlider" name="editSliderCuttingTuneUpPower" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Engraving</string>
           </property>
           <layout class="QFormLayout" name="formLayout_5">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Row spacing</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="EditSlider" name="editSliderEngravingRowSpacing" native="true"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Column Spacing</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="EditSlider" name="editSliderEngravingColumnSpacing" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Engraving tune-up</string>
           </property>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Power</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="EditSlider" name="editSliderEngravingTuneUpPower" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSystem">
      <attribute name="title">
       <string>System</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_6">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Move Start Speed(mm/s)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="EditSlider" name="editSliderMoveStartSpeed" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Move Run Speed(mm/s)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>X Backlash</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Y Backlash</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Total Running Time(Minutes)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Laser Total Use Time(Minutes)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Cut Laser Frequency</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Carving Laser Frequency</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="EditSlider" name="editSliderMoveRunSpeed" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="EditSlider" name="editSliderXBacklash" native="true"/>
       </item>
       <item row="3" column="1">
        <widget class="EditSlider" name="editSliderYBacklash" native="true"/>
       </item>
       <item row="6" column="1">
        <widget class="EditSlider" name="editSliderCutLaserFrequency" native="true"/>
       </item>
       <item row="7" column="1">
        <widget class="EditSlider" name="editSliderCarvingLaserFrequency" native="true"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditTotalRunningTime">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEditLaserTotalUseTime">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabManufactor">
      <attribute name="title">
       <string>Manufacturer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Manufacturer Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditManufacturerPassword"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonVerifyPassword">
           <property name="text">
            <string>Verify</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClearPassword">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonResetPassword">
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxManufactor">
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout_7">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Reset Speed(mm/s)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="EditSlider" name="editSliderResetSpeed" native="true"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Zero Speed(mm/s)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="EditSlider" name="editSliderZeroSpeed" native="true"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Zero Position</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>X Step Length(um)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Y Step Length(um)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>Motor Direction</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Limit Direction</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Page Width(mm)</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Page Height(mm)</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>Drawing Unit(per inch)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="EditSlider" name="editSliderPageWidth" native="true"/>
          </item>
          <item row="8" column="1">
           <widget class="EditSlider" name="editSliderPageHeight" native="true"/>
          </item>
          <item row="9" column="1">
           <widget class="EditSlider" name="editSliderDrawingUnit" native="true"/>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBoxZeroPosition">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxXStepLength">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxYStepLength">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBoxMotorDirection">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="comboBoxLimitDirection">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonRead">
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWrite">
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDefault">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditSlider</class>
   <extends>QWidget</extends>
   <header>widget/LRWidgets/widget/EditSlider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeviceSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeviceSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
